tools/wptserve/wptserve/stash.py

```diff
@@ -147,6 +147,9 @@ def _wrap_key(self, key, path):
         # This key format is required to support using the path. Since the data
         # passed into the stash can be a DictProxy which wouldn't detect changes
         # when writing to a subdict.
+        if isinstance(key, binary_type):
+            # UUIDs are within the ASCII charset.
+            key = key.decode('ascii')
         return (str(path), str(uuid.UUID(key)))
```

Comment:
Can we fix the type here to binary in all cases rather than using `str`?

Reply:
Sure, although this is really internal to Stash. I don't think it's exposed anywhere.

For UUID, if we want bytes, I think we should use its `.bytes` property instead of doing our own encoding.